app/controllers/agenda-items/index.ts

```diff
@@ -173,25 +173,13 @@ const agendaItemsQuery = ({
     'sessions.governing-body.administrative-unit.location',
   ].join(','),
   sort: '-sessions.planned-start',
-  filter: {
-    sessions: {
-      ':gt:planned-start': plannedStartMin ? plannedStartMin : undefined,
-      ':lt:planned-start': plannedStartMax ? plannedStartMax : undefined,
-      'governing-body': {
-        'is-time-specialization-of': {
-          'administrative-unit': {
-            location: {
-              ':id:': locationIds ? locationIds : undefined,
-            },
-          },
-        },
-      },
-    },
-    ':or:': {
-      title: keyword ? keyword : undefined,
-      description: keyword ? keyword : undefined,
-    },
-  },
+  filter: keyword || undefined,
+  'filter[:or:][sessions][governing-body][is-time-specialization-of][administrative-unit][location][:id:]':
```

Comment:
Complex filters are to be string keys because we cannot mix: `filter: keyword || undefined,` and `filter: {` in the same object.